core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -934,7 +934,7 @@ public function loadDetails()
 					{
 						// If $folderUrl is not yet set, use the alias of the first
 						// parent page if it is not a root page (see #2129)
-						if (!$folderUrl && $objParentPage->alias)
+						if (!$folderUrl && $objParentPage->alias && $objParentPage->alias !== 'index' && $objParentPage->alias !== '/')
```

Comment:
When would the parent page alias ever be `/`? You cannot fill `alias` with just `/`.

Reply:
it's a supported use case 🤷. I was surprised by this as well when implementing the new routing, but apparently that did work before as well (even though you maybe only add that in the DB 😂)
https://github.com/contao/contao/blob/5.x/core-bundle/src/Routing/RouteProvider.php#L174

Reply:
> it's a supported use case 🤷

How so? You cannot create a page in the back end with just `/` as an alias.